### PR TITLE
upgrade photos++ to 3.61

### PR DESCRIPTION
**Pull request by mkirsano**

### CODE REVIEW

File: photospp-toolfile.spec [hunk 1]
```diff
@@ -1,4 +1,4 @@
-### RPM external photospp-toolfile 3.56
+### RPM external photospp-toolfile 2.0
```
Comment (iahmad-khan) [inline photospp-toolfile.spec]: @mkirsano the version number is different in spec file than tool-file , shouldn't it be the same?

Comment (mkirsano) [inline photospp-toolfile.spec]: There is no such rule in cmsdist - see other toolfiles. The content of toolfiles is rarely changed.

**Comment by ghost** [comment]
please test

**Comment by davidlange6** [comment]
if we want to change the tool file version for some reason, it should at least increase in number, not go backwards...

File: photospp-toolfile.spec (hunk 1 above)
Comment (smuzaffar) [inline photospp-toolfile.spec]: correct, version in toolfile has nothing to do with version of tool. toolfile version should only be changed when the tool definition is changed

**Comment by smuzaffar** [comment]
agree with @davidlange6 
Although we do not deploy toolfile but it would still be nice to increase the version. I would suggest to set it to 4.0

**Comment by mkirsano** [comment]
I agree with 4.0. Let's wait the test results.

**Comment by mkirsano** [comment]
I think the errors are unrelated - I looked at some of them. I had doubts about compilation, but it is OK.

**Comment by ghost** [comment]
please test

**Comment by ghost** [comment]
@mkirsano Tests are fine. If you could please change the specfile version as discussed then we'll move on and merge this PR.

**Pull request merged**